Source/Core/DolphinQt/Config/ControllersWindow.h

```diff
@@ -9,6 +9,7 @@ class CommonControllersWidget;
 class GamecubeControllersWidget;
 class QDialogButtonBox;
 class WiimoteControllersWidget;
+class MouseSettingsWidget;
```

Comment:
This forward declaration seems unnecessary...?

Reply:
Yeah it was from an old version and I didn't delete it